Makefile

```diff
@@ -32,7 +32,7 @@ apple: $(BUILDDIR)/apple/Tun2socks.xcframework
 $(BUILDDIR)/apple/Tun2socks.xcframework: $(GOMOBILE)
   # MACOSX_DEPLOYMENT_TARGET and -iosversion should match what outline-client supports.
   # TODO(fortuna): -s strips symbols and is obsolete. Why are we using it?
-	export MACOSX_DEPLOYMENT_TARGET=10.14; $(GOBIND) -iosversion=9.0 -target=ios,iossimulator,macos -o $@ -ldflags '-s -w' -bundleid org.outline.tun2socks $(IMPORT_PATH)/outline/apple $(IMPORT_PATH)/outline/shadowsocks
+	export MACOSX_DEPLOYMENT_TARGET=10.14; $(GOBIND) -iosversion=9.0 -target=ios,iossimulator,macos,maccatalyst -o $@ -ldflags '-s -w' -bundleid org.outline.tun2socks $(IMPORT_PATH)/outline/apple $(IMPORT_PATH)/outline/shadowsocks
```

Comment:
The `$(GOBIND)` is not working on my machine because my env var contains space, updating it (@ line 6) to `GOBIND=env PATH="$(GOBIN):$(PATH)" $(GOMOBILE) bind` (just added quotes) will work.

Here is the error message:

```
env: Fusion.app/Contents/Public:/usr/local/go/bin:/Library/Apple/usr/bin: No such file or directory
```

due to this env var:

```
/Applications/VMware Fusion.app/Contents/Public
```

Reply:
Thanks!  I've tried to add quotes around all the places where variables that could potentially contain spaces are passed to the shell.  (This seems to be a known issue with Makefiles.)  I doubt it's 100% correct now but it's probably closer.